### ISSUE TITLE
suggested setting change

### ISSUE THREAD

**Issue by bczretribution** [AaronGullickson/mekhq#46]
### Environment
for MM

### Description
based on my understanding of the balance and style of play you are going for (having only played mission 1- Cutting Teeth so far), would recommend the addition of ending a fight after /victory or /defeat at the end of the phase instead of the end of the turn to the default settings.

Reasoning: Cutting teeth briefing says "The 12th Falcon Regulars should ask for hegira if reduced to less than half their original force size while the player has more BV remaining. This can be simulated in MegaMek with the /victory command."...  such victory likely will be claimed in the initiative phase where force strength and compositions are listed.  However with current settings, that means a full turn of combat will still be fought after victory was achieved.

**Comment by bczretribution** [comment]
To clarify, this option already exists within megamek settings and would just need to be turned on.

**Comment by AaronGullickson** [comment]
Yep, should be straightforward. Waiting on a few other things to make a batch of changes to the campaign file.
